src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/MethodTypeInference.cs

```diff
@@ -1723,6 +1723,11 @@ private bool LowerBoundConstructedInference(TypeSymbol source, TypeSymbol target
                 return false;
             }
 
+            if (constructedTarget.IsTupleType)
+            {
+                constructedTarget = ((TupleTypeSymbol)constructedTarget).UnderlyingTupleType;
```

Comment:
Yes. Looks like we are. 
Inference is mostly NYI. I have added these just to enable some trivial scenarios related to overloading. 
Will add PROTOTYPE(tuples) 

Interesting scenarios may arise if we infer two tuple types differing only in names.